src/buza/admin.py

```diff
@@ -6,3 +6,40 @@
 @admin.register(models.User)
 class UserAdmin(admin.ModelAdmin):
     list_display = ['username', 'school', 'grade']
+
+
+class AnswerInline(admin.TabularInline):
+    extra = 0
+    model = models.Answer
+
+    fields = ['author', 'body']
+    raw_id_fields = ['author']
+
+
+@admin.register(models.Question)
+class QuestionAdmin(admin.ModelAdmin):
+    date_hierarchy = 'created'
+    ordering = ['-created']
+    list_display = ['title', 'author', 'created']
+    search_fields = ['title', 'author__username']
+
+    raw_id_fields = ['author']
+    readonly_fields = ['created', 'modified']
+
+    inlines = [AnswerInline]
+
+
+@admin.register(models.Answer)
+class AnswerAdmin(admin.ModelAdmin):
+    date_hierarchy = 'created'
+    ordering = ['-created']
+    list_display = ['body', 'author', 'question', 'created']
+    search_fields = [
+        'body',
+        'author__username',
```

Comment:
does the  `__` means `.`
for example does `author__username` translate to `author.username`?

Reply:
Yes: the `__` is part of Django's field lookups syntax:

* https://docs.djangoproject.com/en/2.0/topics/db/queries/#field-lookups-intro
* https://docs.djangoproject.com/en/2.0/ref/models/querysets/#id4

These let you traverse relations (like getting the question's author's username), as well as specify other kinds of conditions and comparisons on the fields (like greater than, less than, case-insensitive compare, and so on).